app/assets/stylesheets/components/search.css

```diff
@@ -0,0 +1,7 @@
+form#search .c-loader {
```

Comment:
Black candy basically use ITCSS structure to organize css, you can check article about ITCSS https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/. And we should avoid use id selector in component css, it's hard to maintain